homeassistant/components/kira/remote.py

```diff
@@ -31,32 +31,18 @@ def setup_platform(
         add_entities([KiraRemote(device, kira)])
 
 
-class KiraRemote(Entity):
+class KiraRemote(remote.RemoteEntity):
```

Comment:
Whilst working on type hints, I discovered that `KiraRemote` was implementing `Entity` instead of `RemoteEntity`.
I'm pretty sure that it was a bug.